src/test/resources/features/pre.3.6.x.feature

```diff
@@ -213,17 +211,16 @@ Scenario Outline: <testId>
+|"NASA-PDS/validate#47 1" |"NASA-PDS/validate#47 1" | "github47" | 0 | "0 errors expected." | "CONTEXT_REFERENCE_NOT_FOUND" | "src/test/resources" | "target/test" | "-r {reportDir}/report_github47_disable-valid.json -s json --disable-context-mismatch-warnings -R pds4.label --skip-content-validation --skip-context-validation {resourceDir}/github47/test_context_products.xml" | "report_github47_disable-valid.json" |
+|"NASA-PDS/validate#47 2" |"NASA-PDS/validate#47 2" | "github47" | 1 | "1 errors expected." | "CONTEXT_REFERENCE_NOT_FOUND" | "src/test/resources" | "target/test" | "-v1 --skip-content-validation -r {reportDir}/report_github47_enable-valid.json -s json --disable-context-mismatch-warnings -R pds4.label {resourceDir}/github47/test_context_products.xml" | "report_github47_enable-valid.json" |
 
 | NASA-PDS/validate#62  |"NASA-PDS/validate#62 1" | "github62" | 4 | "4 info.label.context_ref_found info messages expected.\n" | "CONTEXT_REFERENCE_FOUND" | "src/test/resources" | "target/test" | "-v1 -r {reportDir}/report_github62_1.json -s json --no-data-check -t {resourceDir}/github62/ele_mom_tblChar.xml"  | "report_github62_1.json" |
 | NASA-PDS/validate#62-1  |"NASA-PDS/validate#62 2" | "github62" | 8 | "8 info/error messages expected.\n" | "CONTEXT_REFERENCE_FOUND,CONTEXT_REFERENCE_NOT_FOUND" |  "src/test/resources" | "target/test" | "-v1 -r {reportDir}/report_github62_2.json -s json --no-data-check -t {resourceDir}/github62/spacecraft.orex_1.1.xml"  | "report_github62_2.json" |
 
 # Move github278 before github71 as it causing issues
-# FIXME 1058: PDF file streams appear to be empty
-# |"NASA-PDS/validate#278 1" | "github278" | 1 | "1 errors expected for CONTEXT_REFERENCE_NOT_FOUND." | "CONTEXT_REFERENCE_NOT_FOUND" | "src/test/resources" | "target/test" | "-R pds4.label -r {reportDir}/report_github278_label_invalid_cucumber.json -s json {resourceDir}/github278/invalid/trk-2-34-revn-l5_tnf_invalid.xml" | "report_github278_label_invalid_cucumber.json" |
-# |"NASA-PDS/validate#278 2" | "github278" | 0 | "0 errors expected for CONTEXT_REFERENCE_NOT_FOUND." | "CONTEXT_REFERENCE_NOT_FOUND" | "src/test/resources" | "target/test" | "-R pds4.label -r {reportDir}/report_github278_label_valid_cucumber.json -s json {resourceDir}/github278/valid/trk-2-34-revn-l5_tnf.xml" | "report_github278_label_valid_cucumber.json" |
+# By declaring the error type, should allow the zero length PDF file to go unnoticed.
+|"NASA-PDS/validate#278 1" |"NASA-PDS/validate#278 1" | "github278" | 1 | "1 errors expected for CONTEXT_REFERENCE_NOT_FOUND." | "CONTEXT_REFERENCE_NOT_FOUND" | "src/test/resources" | "target/test" | "-r {reportDir}/report_github278_label_invalid_cucumber.json -s json -R pds4.label {resourceDir}/github278/invalid/trk-2-34-revn-l5_tnf_invalid.xml" | "report_github278_label_invalid_cucumber.json" |
```

Comment:
@jordanpadams 

278 is tricky. By declaring the error that are you not expecting it allows cucumber to ignore all other errors. Meh, not so good. In this case, the resulting error is that the PDF files are [zero length which is not a valid PDF file](https://stackoverflow.com/questions/17279712/what-is-the-smallest-possible-valid-pdf). Do we keep this basically flawed approach or do we add minimal valid PDF files?

Reply:
Actually, this is more interesting than just validate. I would also like @nutjob4life opinion on it too. The more I type to hand wave the smoke screen the more I think the 291 byte PDF is the way to go. Maybe have one copy and just point the labels at it if we think that 10K of bytes is going to break our storage.